src/AaveV2PolygonReserveFactorUpdate_20230920/AaveV2PolygonReserveFactorUpdate_20230920.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+
+/*
+   _      ΞΞΞΞ      _
+  /_;-.__ / _\  _.-;_\
+     `-._`'`_/'`.-'
+         `\   /`
+          |  /
+         /-.(
+         \_._\
+          \ \`;
+           > |/
+          / //
+          |//
+          \(\
+           ``
+     defijesus.eth
+*/
+
+pragma solidity ^0.8.0;
+
+import 'aave-helpers/v2-config-engine/AaveV2PayloadPolygon.sol';
+import {AaveV2Polygon, AaveV2PolygonAssets} from 'aave-address-book/AaveV2Polygon.sol';
+
+/**
+ * @dev Update Reserve Factors (RF) on Polygon v2 to encourage users to migrate funds to v3.
+ * @author defijesus.eth - TokenLogic
+ * - Snapshot: Direct-to-AIP Framework
+ * - Discussion: https://governance.aave.com/t/arfc-reserve-factor-updates-polygon-aave-v2/13937/6
+ */
+contract AaveV2PolygonReserveFactorUpdate_20230920 is AaveV2PayloadPolygon {
```

Comment:
Seems like this could just be replaced with an `execute()` call without needing all the features of the config engine in the `AaveV2PayloadPolygon` contract, no?

Reply:
@defijesus hey man you modified an old payload and not this one